project_name/tests/test_sign.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import numpy as np
```

Comment:
Do you see in the test suite here that things are failing, saying `numpy` cannot be found?

You might wonder why it works while using `pcds_conda` but not here.
That's because each of our packages have their own set of requirements. We first test them in isolation - not as part of the conda environment we use on a daily basis.

You can fix this by adding `numpy` as a dependency. We specify dependencies in a couple spots for the package manager pip (ignoring the conda builds for now):

1. For "normal" requirements, required to run a package as a user, these would go in `requirements.txt`. 
2. For requirements that are used _only_ in the test suite, those go in `dev-requirements.txt` ("dev" for "development", of course)

Since numpy is only used in your test suite in this PR, it should go in `dev-requirements.txt`. If you open that file, you'll see that `pytest` is listed there currently, because it's what we use to run the test suite.